util/docker/open3d-gpu/scripts/gce-ubuntu-docker-run.sh

```diff
@@ -49,6 +49,7 @@ GCE_VM_BASE_OS=ubuntu20.04
 GCE_VM_IMAGE_SPEC=(--image-project=ubuntu-os-cloud --image-family=ubuntu-2004-lts)
 GCE_VM_CUSTOM_IMAGE_FAMILY=ubuntu-os-docker-gpu-2004-lts
 VM_IMAGE=open3d-gpu-ci-base-$(date +%Y%m%d)
+GCE_CI_TIMEOUT=720 # Self delete VM after timeout
```

Comment:
Could we add some comments on what does this timeout value control? Is the 720 (second?) sufficient for our purpose?

Reply:
Still testing this, so set the timeout at 720s. Didn't work, so it's actually not ready to be merged.
Finally, I'll set the timeout at 7200s (2 hours).